libreyolo/models/rfdetr/nn.py

```
@@ -567,6 +567,203 @@ def forward(self, x: torch.Tensor, targets=None):
+    depth maps where ``0``/non-finite pixels are invalid.
+    """
+
+    _PATCH_SIZE = 14
+    _POS_ENC_SIZE = 37  # 37 * 14 == 518, DINOv2's pretrained image_size
+    _NUM_WINDOWS = 1
+
+    _TRIM_RATIO = 0.2
+    _GRAD_SCALES = 4
+    _GRAD_WEIGHT = 0.5
+
+    def __init__(
```

Ruff (0.15.15)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)